### PR TITLE
added page for switching between estimators

### PR DESCRIPTION
**Pull request by tumbili**
@jgoppert This is for getting people to know how they can switch estimators.
It would now be nice if we can make links to the documentation of the estimators.
I will do it for EKF2, could you set something up for LPE?

### CODE REVIEW

**Comment by julianoes** [comment]
Cool! Why is the param tied to MC? It could be the same for fixedwing, right?

**Comment by tumbili** [comment]
@julianoes Because until now planes (non VTOL) were always just using the old ekf.
Doesn't really make any sense because VTOL can be used with the other estimators.

File: book/advanced-switching_state_estimators.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+**2. INAV position estimator**
+
+The INAV position estimator is a complementary filter for 3D position and velocity states.
+
+
+**3. LPE position estimator**
+
+The LPE position estimator is an extended kalman filter for 3D position and velocity states.
+
```
Comment (mhkabir) [inline book/advanced-switching_state_estimators.md]: It's not an EKF.

File: book/advanced-switching_state_estimators.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+**4. EKF2 attitude, position and wind states estimator**
+
+EKF2 is an extended kalman filter estimating attitude, 3D position / velocity and wind states.
+
+**4. EKF attitude, position and wind states estimator (depriciated)**
```
Comment (mhkabir) [inline book/advanced-switching_state_estimators.md]: Should be "deprecated"

File: book/advanced-switching_state_estimators.md (hunk 1 above)
Comment (tumbili) [inline book/advanced-switching_state_estimators.md]: @mhkabir Having the accelerometer bias in the body frame actually makes it an EKF.

File: book/advanced-switching_state_estimators.md (hunk 2 above)
Comment (tumbili) [inline book/advanced-switching_state_estimators.md]: Thanks!

**Comment by tumbili** [comment]
@mhkabir Thanks for reviewing, will push it in tomorrow.

File: book/advanced-switching_state_estimators.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Switching State Estimators
+This page shows you which state estimators are available on how you can switch between them.
```
Comment (tumbili) [inline book/advanced-switching_state_estimators.md]: Reminder: Should be "...and how you can..."

**Comment by tumbili** [comment]
Merged!

**Comment by tubeme** [comment]
What is the estimator for VTOL in FW flight?

**Pull request merged**